package.json

```diff
@@ -6,9 +6,14 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
```

Comment:
the branch should be named feature as a functionality. Create new one with proper naming.